common/src/main/scala/explore/schemas/implicits.scala

```diff
@@ -149,5 +153,19 @@ object itcschema {
       def toITCInput: RadialVelocityInput =
         RadialVelocityInput(metersPerSecond = r.rv.value.assign)
     }
+
+    implicit class ScienceDataOps(val s: ObsQueries.ScienceData) extends AnyVal {
+      // From the list of targets selects the ones relevant for ITC
+      def itcTargets: List[ITCTarget] = s.targets.scienceTargets
+        .collect { case (_, s: SiderealTarget) =>
```

Comment:
IIRC, class matches are discouraged, being equivalent to `.isInstaceOf`. I think we can pattern match directly to `SiderealTarget(_, SiderealTracking(_, _, _, _, Some(rv), _), _, _)`.

Reply:
Are you sure? I thought this was kosher

Reply:
A benefit is to combine both `collect` in only one